src/ID3Frames.js

```diff
@@ -545,3 +545,31 @@ module.exports.COMR = {
         return tag
     }
 }
+
+module.exports.GEOB = {
+    create: (data) => {
+        if (!(data instanceof Array)) {
+            data = [data]
+        }
+        const encoding = 3 // UTF-8
+        return Buffer.concat(data.map((geob) => {
+            return new ID3FrameBuilder("GEOB")
+                .appendStaticNumber(encoding)
+                .appendNullTerminatedValue(geob.mimeType ? geob.mimeType : '')
+                .appendNullTerminatedValue(geob.filename ? geob.filename : '', encoding)
+                .appendNullTerminatedValue(geob.description ? geob.description : '', encoding)
```

Comment:
I was checking the documentation and actually the content description should not be optional.

As per:

**...The first two strings may be omitted, leaving only their terminations. There may be more than one "GEOB" frame in each tag, but only one with the same content descriptor.**

Reference: https://id3.org/id3v2.3.0

I would suggest we throw instead of defaulting if no description is provided.

To be fully compliant, we should event check in the the array that there is not twice the same content description.

Finally, I was thinking, we may want to name this property `contentDescription` to be fully aligned with the specification. This has not been done in the past, but this is something we want to do going forward.